docs/Controllers.md

```diff
@@ -0,0 +1,87 @@
+# Controllers
+
+_(c) AMWA 2022, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
+
+## Informative
+
+A Controller is Client software that interacts with the NMOS APIs to discover, connect and manage devices within a networked media system.
+
+* This document includes normative references to be followed when implementing a Controller.
+* This document covers how the Controller interacts with the NMOS APIs only.
+It does not cover other features of the Controller software, such as presentation.
```

Comment:
Should that have its own bullet `* ` or be indented with two spaces to be part of previous bullet?

Reply:
Should be indented.

Reply:
Minor comment: we usually use `-` (rather than `*` for bullets in NMOS specs), although the linter allows both if they used consistently.